modules/data-entry/src/main/frontend/src/questionnaire/SubjectTypeDialog.jsx

```diff
@@ -24,16 +24,16 @@ import withStyles from '@mui/styles/withStyles';
 import QuestionnaireStyle from "./QuestionnaireStyle.jsx";
 
 function SubjectTypeDialog(props) {
-  const { open, onClose, onSubmit, data, isEdit, currentSubjectType, classes } = props;
+  const { open, onClose, onSuccess, data, isEdit, currentSubjectType, classes } = props;
   const initialParent = currentSubjectType?.["@path"].replace("/" + currentSubjectType["@name"], "") || "/SubjectTypes";
   const subjectTypes = !isEdit ? data : data.filter(item => item["jcr:uuid"] != currentSubjectType["jcr:uuid"]);
 
-  const [ label, setLabel ] = useState(isEdit ? currentSubjectType.label : "");
+  const [ label, setLabel ] = useState("");
   const [ parentSubject, setParentSubject ] = useState(initialParent);
```

Comment:
A bit out of scope, but rename to `parent`. `Subject` is incorrect anyway, (we're dealing with subject **types**), but `/SubjectTypes` is neither anyway.

```suggestion
  const [ parent, setParent ] = useState(initialParent);
```